### PR TITLE
Integrate timeline servlet

### PR DESCRIPTION
**Pull request by micaylamendoza**
I integrated the timeline servlet so the dummy data trick nodes are displayed on the timeline page.
Screen Shot 2020-07-18 at 3 02 26 PM

### CODE REVIEW

File: src/main/webapp/trickeditpage.html [hunk 1]
```diff
@@ -10,9 +10,15 @@
       <div class="form-header">
         <h2>Add Trick:</h2>
       </div>
-      <form style="margin:10px;input" action="/timeline_data" method="POST">
+      <form style="margin:10px;input" action="/timeline_data" method="POST" onsubmit="return results();">
```
Comment (CubeOfCheese) [inline src/main/webapp/trickeditpage.html]: Where is the results function?

Comment (CubeOfCheese) [reply src/main/webapp/trickeditpage.html]: There is no js file linked to this html page, so how can it call a js function? I also don't see a function named results() anywhere

File: src/main/webapp/timeline.js [hunk 2]
```diff
@@ -2,4 +2,55 @@ function addNode() {
...
+
+  var trickTitle = document.createTextNode("Trick: ");
+  var dateTitle = document.createTextNode("Date: ");
+  var linkTitle = document.createTextNode("Link: ");
+  var notesTitle = document.createTextNode("Notes: ");
+  var br1 = document.createElement("br");
+  var br2 = document.createElement("br");
+  var br3 = document.createElement("br");
+  var br4 = document.createElement("br");
+  var br5 = document.createElement("br");
```
Comment (CubeOfCheese) [inline src/main/webapp/timeline.js:40]: I think you could declare a single break element and use it multiple times below

File: src/main/webapp/timeline.js [hunk 3]
```diff
@@ -2,4 +2,55 @@ function addNode() {
...
+    const trickNode = document.getElementById('trick-results');
+    tricks.forEach((trick) => {
+      trickNode.appendChild(createTrickElement(trick));
+    })
+  });
+}
+
+function createTrickElement(trick) {
+  const trickElement = document.createElement('div')
+  trickElement.TrickNode = 'trick-container';
```
Comment (CubeOfCheese) [inline src/main/webapp/timeline.js]: what is TrickNode?

File: src/main/webapp/timeline.js [hunk 4]
```diff
@@ -2,4 +2,55 @@ function addNode() {
     var timeline = document.getElementById("timeline");
     console.log(timeline.innerHTML)
     timeline.innerHTML = timeline.innerHTML.concat("<span class=\"timeline-node\"><>--</span>");
-}
\ No newline at end of file
+}
+
+function loadResults() {
+  fetch('/timeline_data').then(response => response.json()).then((tricks) => {
+    const trickNode = document.getElementById('trick-results');
```
Comment (CubeOfCheese) [inline src/main/webapp/timeline.js]: Instead of trick-results, this should be referencing "timeline"

File: src/main/webapp/timeline.html [hunk 5]
```diff
@@ -26,8 +26,12 @@ <h2>Skate Type</h2>
           <p id="trick-name">Trick: </p>
           <p id="date">Date: </p>
           <p id="link">Link: </p>
+          <p id="notes">Notes: </p>
       </div>
-    
+
+      <div id="trick-results"></div>
+      <p id="trick-container"></p>
```
Comment (CubeOfCheese) [inline src/main/webapp/timeline.html]: What is this p element for?

**Review by CubeOfCheese — CHANGES_REQUESTED** [review]

File: src/main/webapp/trickeditpage.html (hunk 1 above)
Comment (micaylamendoza) [reply src/main/webapp/trickeditpage.html]: Oh I forgot to update that. The functions is supposed to be called loadResults()

File: src/main/webapp/timeline.js (hunk 2 above)
Comment (micaylamendoza) [reply src/main/webapp/timeline.js:40]: I originally did that but nothing printed when I tried appending the same element multiple times in different places. When I did this, though, it worked. I can try again to double check

File: src/main/webapp/timeline.js (hunk 4 above)
Comment (micaylamendoza) [reply src/main/webapp/timeline.js]: Thanks!

File: src/main/webapp/timeline.html (hunk 5 above)
Comment (micaylamendoza) [reply src/main/webapp/timeline.html]: Oh I need to remove that line

**Review by CubeOfCheese — APPROVED** [review]

**Pull request merged**